mtp_noms_ops/templates/security/checks_list.html

```diff
@@ -109,9 +109,27 @@ <h1 class="heading-xlarge">{{ view.title }}</h1>
           <td>
             {{ check.credit.amount|currency }}
           </td>
-          <td class="print-hidden">
-            <a href="{% url 'security:resolve_check' check_id=check.id %}">{% trans 'Review' %} <span class="visually-hidden">{% trans 'credit to' %} {{ check.credit.prisoner_name }}</span></a>
+          <td>
+            <a class="govuk-button print-hidden" href="{% url 'security:resolve_check' check_id=check.id %}">{% trans 'Review' %} <span class="visually-hidden">{% trans 'credit to' %} {{ check.credit.prisoner_name }}</span></a>
           </td>
+          {% if view.get_class_name == 'CheckListView' %}
+            {% if check.assigned_to == form.request.user.pk %}
+              <td class="mtp-check-cell__added-to-my-list">
+                Added to my list
+              </td>
+            {% elif check.assigned_to != None %}
+              <td>
+                Added to {{ check.assigned_to_name }}'s List
+              </td>
+            {% else %}
+              <td>
+                <form action="{% url 'security:assign_check' check_id=check.id list=1 %}" method="post">
+                  {% csrf_token %}
+                  <button type="submit" name="assignment" class="govuk-button govuk-button--secondary" value="assign"> {% trans 'Add to my List' %}</button>
```

Comment:
Can we include the prisoner name or other specific reference to the Add to list button, like the Review button has `{% trans 'Review' %} <span class="visually-hidden">{% trans 'credit to' %} {{ check.credit.prisoner_name }}</span>`

Minor point for accessibility reasons, but don't see any harm doing it.